var/spack/repos/builtin/packages/geant4/package.py

```diff
@@ -24,62 +24,80 @@
 ##############################################################################
 
 from spack import *
+import platform
 
 
-class Geant4(Package):
+class Geant4(CMakePackage):
     """Geant4 is a toolkit for the simulation of the passage of particles
     through matter. Its areas of application include high energy, nuclear
     and accelerator physics, as well as studies in medical and space
     science."""
 
     homepage = "http://geant4.cern.ch/"
-    url      = "http://geant4.cern.ch/support/source/geant4.10.01.p03.tar.gz"
+    url = "http://geant4.cern.ch/support/source/geant4.10.01.p03.tar.gz"
 
+    version('10.02.p02', '6aae1d0fc743b0edc358c5c8fbe48657')
     version('10.02.p01', 'b81f7082a15f6a34b720b6f15c6289cfe4ddbbbdcef0dc52719f71fac95f7f1c')
     version('10.01.p03', '4fb4175cc0dabcd517443fbdccd97439')
 
-    variant('qt', default=False, description='Enable Qt support')
+    variant('qt', default=True, description='Enable Qt support')
 
     depends_on('cmake@3.5:', type='build')
 
+    depends_on("clhep@2.3.1.1~cxx11+cxx14", when="@10.02.p02")
     depends_on("clhep@2.3.1.1~cxx11+cxx14", when="@10.02.p01")
     depends_on("clhep@2.2.0.4~cxx11+cxx14", when="@10.01.p03")
     depends_on("expat")
-    depends_on("zlib")
```

Comment:
It may not say so in the documentation, but I've seen a lot of packages that secretly depend on zlib. Can you run `ldd -r` on any shared libraries or binaries that are built and make sure it isn't picking up the system zlib?